CHANGELOG.md

```diff
@@ -18,177 +20,185 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.6.2] - 2024-02-21
+
+### Added
+
+- Unit tests for id3 feature helper functions [#74](https://github.com/streaming-video-technology-alliance/common-media-library/pull/74)
```

Comment:
Unfortunately changing the whitespace in the CHANGELOG will break one of our build scripts, `/scripts/createReleaseNotes.ts`. Either revert all of the white space changes, including the trailing spaces in the links section, or refactor the release notes script to account for the new formatting.

Reply:
Just fixed the requested changes. Now another question came to my mind when running the script is: 
Should we append these updates to version 0.6.2 as an additional note, or would it be better to introduce a new version (0.6.3) for these changes? What do you think about that?

Reply:
Put them in the `Unreleased` section and we will move them in the next release.

Reply:
okay great! I just made the change.